src/main/java/io/jenkins/plugins/pipelinegraphview/PipelineGraphDisplayURLProvider.java

```diff
@@ -2,12 +2,13 @@
 
 import edu.umd.cs.findbugs.annotations.NonNull;
 import hudson.Extension;
-import hudson.Util;
+import hudson.model.Job;
 import hudson.model.Run;
-import org.jenkinsci.plugins.displayurlapi.ClassicDisplayURLProvider;
+import org.jenkinsci.plugins.displayurlapi.DisplayURLProvider;
+import org.jenkinsci.plugins.workflow.job.WorkflowRun;
 
-@Extension
-public class PipelineGraphDisplayURLProvider extends ClassicDisplayURLProvider {
+@Extension(ordinal = 50) // Take precedence over Blue Ocean if both are installed and there is no configured provider.
+public class PipelineGraphDisplayURLProvider extends DisplayURLProvider {
```

Comment:
The upstream behavior is weird, basically all implementations of `ClassicDisplayURLProvider` get ignored when picking the default provider if there is no specific configuration. Switching to `DisplayURLProvider` allows this provider to be selected automatically. (If you do not want this, we could revert it, but this is how `blueocean-display-url` is able to activate automatically.)